src/CMakeLists.txt

```diff
@@ -46,6 +46,13 @@ else()
     endif()
   endforeach()
 
+  #Fix missing kernels in split binaries dependency
+  list(APPEND ATen_XPU_SYCL_UNARY_BINARY_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/ATen/native/xpu/sycl/CopyKernel.cpp")
+
+  list(APPEND ATen_XPU_SYCL_NON_UNARY_BINARY_SRCS 
+      "${CMAKE_CURRENT_SOURCE_DIR}/ATen/native/xpu/sycl/UnaryFractionKernels.cpp" 
+      "${CMAKE_CURRENT_SOURCE_DIR}/ATen/native/xpu/sycl/UnaryKernels.cpp")
+
```

Comment:
Are there duplicated implementations both in unary and non unary lib? Could you try,
<img width="734" alt="image" src="https://github.com/user-attachments/assets/8cb21c51-eaae-47b3-a92f-c730f342db3c">